code/web/src/modules/user/Login.js

```diff
@@ -47,13 +49,19 @@ class Login extends Component {
     })
   }
 
+// This method is what handles the event of someone submitting the form. This invoked whenever a user
+// clicks enter or the 'Login' button.
   onSubmit = (event) => {
     event.preventDefault()
 
     this.props.messageShow('Logging in, please wait...')
 
+// This login method here is one of our actions from in ./api/actions.js. This action is where
+// our axios fetch request lives to grab a specific users data from the api based on the login credentials
+// which then updates our user property of our applications state.
     this.props.login(this.state.user)
```

Comment:
Is the login prop just an object containing authentication info?  Would helpers involving this be added to the prop (which is how models are treated in rails) or elsewhere?